docs/overview/index.md

```diff
@@ -1,17 +1,105 @@
+
+- **General decentralized computing**: Decentralized computing allows developers to write programs called _smart contracts_ and be assured that these programs will run as intended without censorship.
+
 ## Cutting-edge developments on Tezos
 
-With recent Tezos upgrades heavily focusing on scaling solutions, Tezos has been at the forefront of blockchain research. Up to 1 million transactions per second (TPS) are theoretically possible on Tezos with the advent of Smart Rollups, which have emerged as a strong scaling solution. With Smart Rollups (also known as optimistic rollups), transactions can be performed more quickly and cheaply and use other VMs that “roll up” their state changes onto Tezos.
+With recent Tezos upgrades heavily focusing on scaling solutions, Tezos has been at the forefront of blockchain research. Up to 1 million transactions per second (TPS) are theoretically possible on Tezos with the advent of [Smart Rollups](../architecture/smart-rollups), which have emerged as a strong scaling solution. With Smart Rollups (also known as optimistic rollups), transactions can be performed more quickly and cheaply and use other VMs that “roll up” their state changes onto Tezos.
 
-[Etherlink](https://www.etherlink.com) is another exciting technology in development that will greatly enlarge the possible applications and versatility of Tezos. Etherlink builds on Smart Rollups and combines them with [DAL (Data Availability Layer)](https://research-development.nomadic-labs.com/dal-testers.html) to provide an EVM-compatible Layer 2 solution which is truly decentralized, low-latency, and low-cost.
+[Etherlink](https://www.etherlink.com) is another exciting technology in development that will greatly enlarge the possible applications and versatility of Tezos.
+Etherlink builds on Smart Rollups and combines them with the [Data Availability Layer](/architecture/data-availability-layer) to provide an EVM-compatible Layer 2 solution which is truly decentralized, low-latency, and low-cost.
 
 These innovations demonstrate the ongoing dedication of Tezos to foster innovation in the blockchain industry.
+
+## What are some example applications of Tezos?
+
+Here are some of the ways that people use Tezos:
+
+### NFTs on Tezos
+
+The rise of non-fungible tokens (NFTs) has become a core arena of activity and growth in the blockchain space, where new communities are growing around artists, musicians, and other creators with the use of blockchain tokens. NFTs allow collectors and fans to hold a piece of content produced by a creator, providing proof of ownership and secure exchanges on the blockchain.
+
+[Objkt](https://objkt.com/), [FxHash](https://www.fxhash.xyz/), [Teia](https://teia.art/), [DNS.xyz](https://dns.xyz/) are some of the marketplaces for Tezos art and music NFTs. They allow creators to mint and sell their work in addition to providing a platform for users to collect such works.
```

Comment:
"in addition to" seems incorrect, because minting and collecting are pointless without each other, so there are no two different things, but just two faces of the same thing - one for authors and one for users.
Replace by "and provide" or "and allow users"?

Reply:
Changed to "They allow creators to mint and sell their work in addition to providing a platform for users to collect such works."

Reply:
Changed to "They provide a platform for creators to mint and sell their work and for users to collect such works."